helm/bankapp/templates/service.yml

```
@@ -0,0 +1,16 @@
+metadata:
+  name: "{{ .Values.app_deployment.name }}-service"
+  namespace: {{ default "bankapp-namespace" .Values.namespace }}
+  labels:
+    app: {{ .Values.app_deployment.name }}
+spec:
+  type: NodePort
+  selector:
+    app: {{ .Values.app_deployment.name }}
+  ports:
+  - protocol: TCP
+    port: {{ default "8080" .Values.bankapp_svc.port}} 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)